src/Metalama.Framework.RunTime/Metalama.Framework.RunTime.csproj

```
@@ -6,6 +6,10 @@
         <Nullable>enable</Nullable>
         <GenerateDocumentationFile>true</GenerateDocumentationFile>
         <PackageReadmeFile>README.md</PackageReadmeFile>
+        <PackageDescription>
+            Run-time components for package 'Metalama.Framework'. This package should usually only be installed as a dependency.
+            (This is not the package you are looking for, unless you've performed `metalama divorce`.)
```

```suggestion
            (To use Metalama in your project, install the 'Metalama.Framework' package instead.)
```

I'd use this formulation. If someone uses the divorce feature, he should know already that this package is needed.